### PR TITLE
Add interface metrics

### PR DESCRIPTION
**Pull request by bluecmd**
Adds:

 * `fortigate_interface_link_up`
 * ~`fortigate_interface_speed`~ `fortigate_interface_speed_bps`
 * `fortigate_interface_transmit_packets_total`
 * `fortigate_interface_receive_packets_total`
 * `fortigate_interface_transmit_bytes_total`
 * `fortigate_interface_receive_bytes_total`
 * `fortigate_interface_transmit_errors_total`
 * `fortigate_interface_receive_errors_total`

### CODE REVIEW

**Comment by bluecmd** [comment]
Ping @SuperQ - what do you think about the metric names?

File: README.md [hunk 1]
```diff
@@ -26,6 +26,14 @@ Per-VDOM:
  * `fortigate_policy_bytes_total`
  * `fortigate_policy_hit_count_total`
  * `fortigate_policy_packets_total`
+ * `fortigate_interface_link_up`
+ * `fortigate_interface_speed`
```
Comment (SuperQ) [inline README.md:30]:
```suggestion
 * `fortigate_interface_speed_bps`
```

File: probe.go [hunk 2]
```diff
@@ -335,6 +335,119 @@ func probeFirewallPolicies(c FortiHTTP, registry *prometheus.Registry) bool {
...
+		mLink = prometheus.NewGaugeVec(
+			prometheus.GaugeOpts{
+				Name: "fortigate_interface_link_up",
+				Help: "Whether the link is up or not",
+			},
+			[]string{"vdom", "name", "alias", "parent"},
+		)
+		mSpeed = prometheus.NewGaugeVec(
+			prometheus.GaugeOpts{
+				Name: "fortigate_interface_speed",
```
Comment (SuperQ) [inline probe.go:349]:
```suggestion
				Name: "fortigate_interface_speed_bps",
```

**Comment by bluecmd** [comment]
Had to create a new commit to include tests as well, added you as co-author :-)

**Pull request merged**